app/services/exports/case_log_export_service.rb

```diff
@@ -181,15 +181,82 @@ def apply_cds_transformation(case_log, export_mode)
       attribute_hash["uploaddate"] = attribute_hash["updated_at"]
       attribute_hash["tenancycode"] = attribute_hash["tenant_code"]
       attribute_hash["ppcodenk"] = attribute_hash["previous_postcode_known"]
+      attribute_hash["sheltered"] = attribute_hash["shelteredaccom"]
 
       # Age refused
       (1..8).each do |index|
         attribute_hash["age#{index}"] = -9 if attribute_hash["age#{index}_known"] == 1
       end
 
+      attribute_hash["hhtype"] = get_hhtype(case_log)
+      attribute_hash["new_old"] = get_new_old(case_log)
+      attribute_hash["vacdays"] = get_vacdays(case_log)
+
       attribute_hash
     end
 
+    def get_vacdays(case_log)
```

Comment:
Personal preference rather than anything that needs changing but I think nouns (what it returns) over verbs for method names is more idiomatic ruby (i.e. this would be `vacdays` rather than `get_vacdays`.